src/main/java/at/fh/joanneum/irfc/rest/EventInfoApi.java

```diff
@@ -0,0 +1,55 @@
+package at.fh.joanneum.irfc.rest;
+
+import at.fh.joanneum.irfc.model.eventInfo.EventInfoDTO;
+import at.fh.joanneum.irfc.service.EventInfoService;
+
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ * @author gregor.wakonig@edu.fh-joanneum.at
+ **/
+@RequestScoped
+@Path("/eventInfos")
+public class EventInfoApi {
+
+    @Inject
+    EventInfoService eventInfoService;
+
+    @GET
+    @Path("/{id}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public EventInfoDTO get(@PathParam("id") Long id){
+        return eventInfoService.get(id);
+    }
+
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    public EventInfoDTO create(EventInfoDTO eventDTOCreate){
```

Comment:
For the EventInfoApi, we will only have the get call public, all other calls will get called only inside the backend (create, update, delte will get called from the eventHandler) => see puml what is private and what not
@PhilipMoser or did I get that wrong?

Reply:
In principle you are right, but now that they are created, I would leave them there. we can always delete them later, when we see that we really don't need them